docs/markdown/snippets/add_compcert_compiler.md

```diff
@@ -0,0 +1,3 @@
+## Added CompCert C compiler
+
+Added support for the CompCert formally-verified C compiler (https://github.com/AbsInt/CompCert).
```

Comment:
This should use the markdown format instead:

    Added support for the [CompCert formally-verified C compiler](https://github.com/AbsInt/CompCert).

Also, it would be useful if there were a couple of sentences describing the state of the implementation, possible gotchas (if any) and so on.

Reply:
State/gotchas of the compiler support implementation inside meson, or state/gotchas of the actual compiler? The `add_nvidia_hpc_sdk_compilers.md` isn't very verbose, either, so I sticked to that non-verbosity ;) But could add a blurb if that's preferred.

Reply:
It's more of a general "if you use this, expect roughly this sort of quality". So whether the implementation is "thoroughly tested with a large set of programs", "fairly good, should be expected to work on most projects" or "somewhat experimental, works with some projects but may have issues".

Reply:
Thanks for the clarification. The snippet has been updated.